ISISComputingGroup/EPICS-inst_servers
# Fixed unit tests

### To test

Confirm that the unit tests now work (broken in https://github.com/ISISComputingGroup/EPICS-inst_servers/pull/89)

---

#### Code Review


### Functional Tests


### Final steps
